Bottom: 563ad345d4a7fd8f2cc3e401dad3fe8c25e91899
Top:    9ead5574030f1a8b3f7a02d5b457e92daab4be3c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:56:26 +0000

Refresh of apply-the-solder-mask-on-top

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index e939b5a..dc7303b 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -606,7 +606,7 @@ object3d_from_board_outline (void)
                                     top_bot_appearance);
 #else
   objects = object3d_from_contours (board_outline,
-                                     HACK_BOARD_THICKNESS / 2, /* Bottom *//
+                                     HACK_BOARD_THICKNESS / 2, /* Bottom */
                                     -HACK_BOARD_THICKNESS / 2, /* Top */
                                     board_appearance,
                                     top_bot_appearance);
